dash-pipeline/bmv2/dash_pipeline.p4

```diff
@@ -454,8 +473,9 @@ control dash_ingress(
 
         src_tag.apply();
         dst_tag.apply();
-
-
+        // #ifdef UNDERLAY
+        nhop.apply();
```

Comment:
make: *** [Makefile:126: p4c-dpdk-pna] Error 1

Reply:
Caused by `#ifdef TARGET_BMV2_V1MODEL` around nhop table - would need to surround the `appply()` as well, although I don't think this should be conditional at all.

Reply:
> Caused by `#ifdef TARGET_BMV2_V1MODEL` around nhop table - would need to surround the `appply()` as well, although I don't think this should be conditional at all.

Yeah, the error was due to #ifdef. I fixed it.

Reply:
Please reconsider if you need any ifdef at all - it should work on either target, so ifdef might not be needed.